projects/client/i18n/meta/en.json

```diff
@@ -1188,24 +1167,23 @@
       "default": "Lists",
       "description": "Title for the lists page.",
       "exclude": [
-        "android",
         "ios"
       ]
     },
     "list_title_watchlist_shows": {
-      "default": "Your Shows",
+      "default": "Shows Watchlist",
       "description": "Title for lists containing shows in the user's watchlist. This title should be as short and concise as possible.",
       "exclude": [
-        "android",
-        "ios"
+        "ios",
+        "web"
       ]
     },
     "list_title_watchlist_movies": {
-      "default": "Your Movies",
+      "default": "Movies Watchlist",
       "description": "Title for lists containing movies in the user's watchlist. This title should be as short and concise as possible.",
       "exclude": [
-        "android",
-        "ios"
+        "ios",
+        "web"
```

Comment:
@seferturan please make sure @anodpixels updates the designs to have the coalesced Lists (and all other sections too).